client/packages/system/src/Patient/JsonForms/useZodOptionsValidation.ts

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
+import { z, ZodError, ZodIssue } from 'zod';
+
+const formatMessage = (issue: ZodIssue): string => {
+  switch (issue.code) {
+    case 'invalid_type': {
```

Comment:
better to use the enum I think:
```suggestion
    case ZodIssueCode.invalid_type: {
```